php/class-customize-snapshot-manager.php

```diff
@@ -634,16 +634,25 @@ public function enqueue_controls_scripts() {
 
 		wp_enqueue_style( 'customize-snapshots' );
 		wp_enqueue_script( 'customize-snapshots' );
+		if ( $this->snapshot ) {
+			$post = $this->snapshot->post();
+			$this->override_post_date_default_data( $post );
+		}
 
 		// Script data array.
 		$exports = apply_filters( 'customize_snapshots_export_data', array(
 			'action' => self::AJAX_ACTION,
 			'uuid' => $this->snapshot ? $this->snapshot->uuid() : self::generate_uuid(),
-			'editLink' => $this->snapshot ? get_edit_post_link( $this->snapshot->post(), 'raw' ) : '',
+			'editLink' => $this->snapshot ? get_edit_post_link( $post, 'raw' ) : '',
```

Comment:
Warning: `$post` may not have been defined. Above you should do `$post = null` to initialize, or else we'll get undefined variable warnings.